stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.3.3] - Oct 15, 2020
+* Fix PodDisruptionBudget selector issue.
```

Comment:
@rahulsadanandan  can you add small description of what the issue is ?

Reply:
Its done now